### ISSUE TITLE
No WiFi on a pi zero 2w after boot up from fresh installation

### ISSUE THREAD

**Issue by Vannue007** [VirusPilot/stratux-pi4#19]
I’ve a working stratux setup with a Pi4 and was trying to make the setup more compact with a pi zero 2w setup.
But I don’t have No WiFi on pi zero 2w after boot up from fresh successfull installation (without any screen or gps module plugged in) is this necessary? I just wanted to first install the pi zero 2w before continuing.

**Comment by VirusPilot** [comment]
I am not fully following what you want to achieve, what do you want to be different on the Pi Zero?

**Comment by Vannue007** [comment]
So for clarification. We have a working setup fixed in the plane and I was building up a new pi zero 2w from scratch “Offline”, at home to replace the current pi4 with an older version of stratux to reduce the power load, so at home it’s without the sdr’s, GPS, as these are in the hangar with the plane and we don’t have it double.

During the build you state that “ after reboot please reconnect LAN and/or WiFi and Stratux should work right away” but then I don’t have a WiFi signal, when I connect a screen to the pi zero, I do see that the system has booted and is working. How can we enable the WiFi again? It looks like it’s disabled after the booting

**Comment by VirusPilot** [comment]
Ah, I see. I had tested my script with a Pi Zero 2W a while ago - let me try again and I will come back with my results.

**Comment by VirusPilot** [comment]
So I just finalized testing my script on a Pi Zero 2W and it works as expected and in particular WiFi is on and working. Please make sure that you follow all steps as described in my Readme:

- flash latest 64bit RasPiOS Lite Image, using latest Raspberry Pi Imager with the following settings:
- select appropriate hostname
- enable ssh
- enable user pi with password
- Pi Zero 2W: enable and configure WiFi so that the Pi connects to your local network
- boot your Pi with this image and wait until it is connected to your LAN or WiFi (this may take a few minutes after the first boot)

after running my script:

- after reboot Stratux is providing an unprotected WiFi access point with the SSID "stratux"
- after you have connected with this access point, you can open the Stratux web interface: http://192.168.10.1/ and modify the settings according to your needs
- Pi Zero 2W users: Stratux will now no longer connect to your local network unless you change this by switching the Stratux WiFi settings to "AP+Client"

**Comment by Vannue007** [comment]
I’ve tried this with 2 new pi zero 2w’s with 2 different sd cards and with the “standard” & “viruspilot” script without success.
When I reboot I still don’t get a “stratux” broadcast but I can see that the ip address is set correct.
![IMG_1561](https://github.com/VirusPilot/stratux-pi4/assets/141065564/3129246e-e717-451a-9ae5-e95fdc6144a4)
![IMG_1560](https://github.com/VirusPilot/stratux-pi4/assets/141065564/a35435cf-1b92-4c81-8f70-9e83303dc689)

can you show the result if you type “iwconfig”? Just for troubleshooting. And do you have sdr’s connected or no peripherals like me?

thank you already for the help

**Comment by VirusPilot** [comment]
Correct, nothing connected:

pi@stratux-pi-zero:~ $ iwconfig
lo        no wireless extensions.

wlan0     IEEE 802.11  ESSID:off/any  
          Mode:Managed  Access Point: Not-Associated   Tx-Power=31 dBm   
          Retry short limit:7   RTS thr:off   Fragment thr:off
          Power Management:off
          
ap0       IEEE 802.11  Mode:Master  Tx-Power=31 dBm   
          Retry short limit:7   RTS thr:off   Fragment thr:off
          Power Management:on

**Comment by VirusPilot** [comment]
This is how imager settings should look like (could not easily switch to english):

![Bildschirmfoto 2023-11-17 um 10 43 10](https://github.com/VirusPilot/stratux-pi4/assets/43483458/3f55b7e5-31bb-44d7-9116-e6dc3942bea5)

**Comment by Vannue007** [comment]
I’ve update the raspberry pi imager from version 1.75 to 1.80 and now it’s working.
Thank you for the assistance on finding a solution!

**Comment by VirusPilot** [comment]
great, good to hear!
